src/setupTests.js

```diff
@@ -3,3 +3,53 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom';
+import { MemoryRouter } from 'react-router-dom';
+import { MockedProvider } from '@apollo/client/testing';
+import { render } from '@testing-library/react';
+import { mockApolloData } from './queries/MockApolloData';
+
+Object.defineProperty(window, 'matchMedia', {
```

Comment:
I'd suggest use `Object.defineProperties`, this will allow you to set all props for window in 1 go.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperties